paths/reset/01-reset-intro.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Introduction to Git Reset
+permalink: /reset/01
+next-page: /reset/02
+facilitator: false
+sidebar:
+  nav: "reset"
+main-content: |
+  When working with Git, you might find yourself in a position where the things you have been working on don't meet your initial needs or aren't working. The `git reset` command allows you to 'reset' your repository (and your Staging Area / Working Directory) to a specific moment in time.
+
+  In this course we discuss three distinct modes of the `reset` command:
+
+  - `--soft`
+  - `--mixed`
+  - `--hard`
+show-me-how:
+tell-me-why:
+   includes: tell-me-why/two-stage-commit.md 
```

Comment:
@beardofedu What do you think of this current 'tell me why'? To me it doesn't seem to quite fit. I'm happy to ✏️  give it a go if you'd like.